compiler/src/ct_checker_forward.ml

```diff
@@ -447,11 +447,11 @@ let get_annot ensure_annot f =
   let check_lvl lvl =
     match lvl with
     | Poly s ->
-      let diff = Svl.diff known s in
```

Comment:
Can you please show a test case that illustrate the wrong behavior (without this change)?

Reply:
Done. It's easy to reproduce, you just need to introduce a security variable in the arguments and not to use it in the return type.